doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -716,7 +716,7 @@ bash configure --openjdk-target=${OPENJ9_CC_PREFIX} \
 
   where:
 
-  - `path_to_library` uses a custom OpenSSL v1.1.x library that's already built.
+  - `path_to_library` uses an OpenSSL v1.1.x library that's already built.  You can use `${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/libc/usr` as `path_to_library` when you are configuring in the Docker container.
```

Comment:
The same change should be added for other versions (8, 12) as well.

Reply:
The other versions don't have the `AArch64` section in the Build Instruction yet.  So this is only for 11 right now.